spec/rails_helper.rb

```diff
@@ -26,7 +27,7 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
+Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |f| require f }
```

Comment:
What profit does provide you method `.sort` ?

Reply:
Lint/NonDeterministicRequireOrder: Sort files before requiring them.

Reply:
ha, ok